### ISSUE TITLE
[fix] pfelement | Use standard `this.isConnected` instead of custom `this.connected`

### ISSUE THREAD

**Issue by mwcz** [patternfly/patternfly-elements#1077]
What I added `this.connected`, I wasn't aware there was already a standardized `this.isConnected`.

The work involves updating `elements/pfelement/src/pfelement.js` to remove all the `this.connected` assignments, and finding anywhere else `this.connected` is referenced and changing them all to reference `this.isConnected` instead.

**Comment by bennypowers** [comment]
I ran a search through the repo but didn't find any instances of `.connected` asides from this event, neither did i find any instances of `.isConnected`

```
$ rg -e '\.connected\b'
elements/pfe-avatar/src/pfe-avatar.js
94:    this.emitEvent(PfeAvatar.events.connected, {
167:      this.addEventListener(PfeAvatar.events.connected, () => this.update());
$ rg -e '\.isConnected\b'
$
```

**Comment by mwcz** [comment]
Good catch @bennypowers, thanks.  It looks like this was fixed by #1091, here's [the rather lengthy squashed commit](https://github.com/patternfly/patternfly-elements/commit/e86f33e0342933f1992d52a022f9a25fd1e2fbeb).
